src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,51 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  margin: 50px auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector. 
DON'T FORGET TO PROOFREAD YOUR CODE WITH [CHECKLIST](https://github.com/mate-academy/layout_stop-watch/blob/master/checklist.md) BEFORE SENDING YOUR PULL REQUEST